pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -45,8 +47,41 @@ import (
 	registryconfig "github.com/openshift/origin/pkg/dockerregistry/server/configuration"
 )
 
+var prune = flag.Bool("prune", false, "Prune blobs from the storage. DANGEROUS! Do it only in read only mode")
```

Comment:
We can, but only if `dockerregistry -prune` is executed in dc/docker-registry's container.